worker/draft.go

```diff
@@ -1760,6 +1760,12 @@ func (n *node) InitAndStartNode() {
 					n.Connect(id, m.Addr)
 				}
 			}
+			for _, id := range sp.Metadata.ConfState.Learners {
+				m, ok := members[id]
```

Comment:
I am not sure why this check is necessary. The loop above had the check and so I've added it.

Reply:
I think this might be needed because of the following reason. Not sure though.
Say an alpha node has a stale snapshot and the cluster has progressed (some nodes added/deleted). When we restart that node, it first gets the snapshot from the WAL and verifies that it actually exists  in latest membership state. If yes, then it connects to it.

This change will also mean that learner nodes will connect among themselves. Do we want that?

Reply:
>This change will also mean that learner nodes will connect among themselves. Is that expected behaviour?

That's expected. All alphas connect to all other alphas.